docs/configuration.md

```diff
@@ -392,6 +392,17 @@ Apart from these, the following properties are also available, and may be useful
   </td>
 </tr>
 <tr>
+  <td><code>spark.reducer.maxReqsInFlight</code></td>
+  <td>20</td>
+  <td>
+    spark.reducer.maxMbInFlight puts a bound on the in flight data in terms of size.
```

Comment:
The full description describes about the configuration if you can take a look at it. The first line just explains why the maxMbInFlight is insufficient